### PR TITLE
Add update-readmes pipeline

### PR DESCRIPTION
**Pull request by MichaelSimons**
Related to #331

I will port the common infra back to docker-tools and dotnet-framework-docker once merged.

### CODE REVIEW

File: eng/pipelines/update-readmes.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+trigger: none
+pr: none
+variables:
+- group: DotNet-Maestro
+- template: variables/common.yml
+- name: manifest
```
Comment (mthalman) [inline eng/pipelines/update-readmes.yml]: Should this be defined as a variable in the pipeline UI so that we can override it for samples perhaps?

Comment (MichaelSimons) [reply eng/pipelines/update-readmes.yml]: I was planning on doing that and leaving this here as a default.  Would that be confusing?

Comment (mthalman) [reply eng/pipelines/update-readmes.yml]: Does the UI actually take precedence over this value then?

Comment (MichaelSimons) [reply eng/pipelines/update-readmes.yml]: yes

**Pull request merged**